examples/debian_snapshot/test_linux_amd64.yaml

```diff
@@ -19,10 +19,22 @@ commandTests:
       - perl/now 5\.32\.1-4\+deb11u3 amd64 \[installed,local\]
       - tzdata/now 2024a-0\+deb11u1 all \[installed,local\]
       - nvidia-kernel-common/now 20151021\+13 amd64 \[installed,local\]
+  - name: "stat dpkg status"
```

Comment:
Please `cp examples/debian_snapshot/test_linux* e2e/smoke/` so the `e2e` tests are in-sync with `debian_snapshot`. Thanks!